terraform/custom_domains/environment_domains/workspace_variables/register_production.tfvars.json

```diff
@@ -0,0 +1,52 @@
+{
+  "multiple_hosted_zones": true,
+  "enable_alerting": true,
+  "pg_actiongroup_name": "s189p01-att-production-ag",
+  "pg_actiongroup_rg": "s189p01-att-pd-rg",
+  "latency_threshold": 5000,
+  "hosted_zone": {
+    "register-trainee-teachers.service.gov.uk": {
+      "front_door_name": "s189p01-register-svc-domains-fd",
+      "resource_group_name": "s189p01-registerdomains-rg",
+      "domains": [
+        "www2"
+      ],
+      "cached_paths": ["/packs/*"],
+      "environment_short": "pd",
+      "environment_tag": "pd",
+      "origin_hostname": "register-production.teacherservices.cloud",
+      "null_host_header": true,
+      "cnames": {
+        "www": {
+        "target": "d3ikbzozms3bbf.cloudfront.net"
+        },
+        "_1191f61775c66c9eec91175aa294cdbb.www": {
+        "target": "_d0ee22f81c78ffe76df87d4ffa134644.mybbdzzyvz.acm-validations.aws.",
+        "ttl": 86400
+        }
+      }
+    },
+    "register-trainee-teachers.education.gov.uk": {
+        "front_door_name": "s189p01-register-edu-domains-fd",
+        "resource_group_name": "s189p01-registerdomains-rg",
+        "domains": [
+          "www2"
+        ],
+        "cached_paths": ["/packs/*"],
+        "environment_short": "pd",
+        "environment_tag": "pd",
+        "origin_hostname": "register-production.teacherservices.cloud",
+        "null_host_header": true,
+        "cnames": {
+          "www": {
+            "target": "d3kffbwt0ldx12.cloudfront.net",
```

Comment:
Is the target for www. different between the education and service domains?

Reply:
Looks like it